data/modules/Scout/ScanManager.lua

```diff
@@ -100,7 +100,7 @@ end
 
 -- Update different scan types at different rates
 local SURFACE_SCAN_UPDATE_RATE = 1
-local ORBITAL_SCAN_UPDATE_RATE = 60
+local ORBITAL_SCAN_UPDATE_RATE = 1
```

Comment:
Note that this is a game-time duration of a scan "tick". At a typical timewarp rate of 100x, this means the tick function is attempting to fire every 1/100th of a second (and being unable to do so, is instead executing every single frame).

Reply:
Leaving it at the original 60 meant that the display is updating very slowly - and part of the various changes was ensuring that the player would get more immediate feedback as to progress.

Is the timewarp rate available? If so we could scale the update rate according to timewarp..

Reply:
.. so to answer myself, yes, there is. However there's no event that's emitted when time acceleration is changed. Is this simply because there hasn't been a need, or is there a reason?

Reply:
Hasn't been a need historically. I'm strongly thinking of extending the LuaTimer API to support queuing a "timewarp invariant" timer (which stops ticking when the game is paused, of course). This would solve this (and other!) issues with timers-under-timewarp.

Reply:
.. so what's the way forward here, should I revert this change (at the cost of significantly delayed UI updates at lower timwarps) or should we hold this PR until the "timewarp-invariant-timer" is implemented? Or leave it as-is and add a TODO comment to fix this?

Reply:
> or should we hold this PR until the "timewarp-invariant-timer" is implemented?

My general thought (not code-review): as long as this PR is a net improvement, we should be wary to stall it / feature creep it further, as it might end up bit-rotting for years.

Reply:
Leave the change as is, potentially adding a TODO comment to it. I'll look at adding a timewarp-invariant timer API in a follow-on PR, but the performance cost of scanning is not a pressing concern worth delaying the PR over.

Reply:
Fixed.  No commit ref as I'm squashing-and-rebasing to hopefully have this PR finalised :)